twisted_s3/client.py

```diff
@@ -53,10 +53,17 @@ def _make_request(
 
         hashed_payload = auth.compute_hashed_payload(payload)
 
-        host = "{bucket}.s3-{region}.amazonaws.com".format(
-            bucket=bucket,
-            region=region
-        )
+        host = ""
+        if region == "us-east-1":
+            host = "{bucket}.s3.amazonaws.com".format(
```

Comment:
Have you tried this host format for other regions? Could it just work for all of them, and we could just stick with this?